src/hashcash.cr

```diff
@@ -4,37 +4,52 @@ require "base64"
 require "digest/sha1"
 
 module Hashcash
-  VERSION = "0.1.0"
+  VERSION       = "0.1.0"
+  STAMP_VERSION = 1
+
+  # Hashcash.generate("resource")
+  # => 1:20:201206222555:resource::pOWgc88+uDuefr/o:MTMxNzg2MA==
+  # OR can customise hashcash defaults
+  # Hashcash.generate("hello", version: 1, bits: 16, date: Time.utc, ext: "goodbyye")
+  # => 1:16:201206222403:hello:goodbyye:kfwaGRadlD3ddc9G:MTMxMzY5NQ==
+  def self.generate(
+    resource : String, *,
```

Comment:
What's the capture all (`*`) param here for?

Reply:
was trying something, not needed I'm pretty sure